netlify.toml

```diff
@@ -1,4 +1,10 @@
+[dev]
+  framework = "vite"
 [build]
+  framework = "vite"
   command = "npm run build"
   publish = "build/"
   functions = "functions/"
+[[redirects]]
+  from = "/js/*"
```

Comment:
what is this for?

Reply:
The redirect is just to shorten the url (instead of `/.netlify/functions/attendance`, can access `/js/attendance`)😅